### ISSUE TITLE
Hosted sites are down

### ISSUE THREAD

**Issue by TysonMN** [CONNECT-platform/coding-blog#60]
For example, https://eugene.coding.blog/ is down.

**Comment by im-coder-lg** [comment]
Same :/

**Comment by loreanvictor** [comment]
Yes. I suspect we should switch to another solution. I would recommend deploying blogs on GH-pages for now, and I will figure out how to redirect domains manually to blogs that are up for the moment at least.

**Comment by loreanvictor** [comment]
But the build / hosting infra I think is getting deprecated and it is too costly to maintain it due to low traction.

**Comment by loreanvictor** [comment]
@TysonMN @im-coder-lg So I've brought [eugene.coding.blog](https://eugene.coding.blog) back up, simply by hosting it using GH-pages and pointing the domain to the repository. You can take a look at the [repo](https://github.com/loreanvictor/techblog), specifically the corresponding [CODEDOC config](https://github.com/loreanvictor/techblog/blob/master/.codedoc/config.ts), and setup your repositories accordingly, having GH pages serve your blogs. I can then connect corresponding coding blog subdomains to your blogs, effectively bringing your blogs back online.

This process of course is not an optimal one (since I manually need to connect every single blog), but works for now. Generally, I think we should move coding blog towards a simple domain name where x.coding.blog is a nice programming blog (not necessarily using CODEDOC even, anything that can be hosted via GitHub pages would work). I would need to update the website accordingly and setup proper scripts so people can basically create PRs on coding.blog repository and have their domains connected to their repositories, but for now the manual process should work.

**Comment by im-coder-lg** [comment]
I'll do this later, feeling sick 🤧 rn

**Comment by loreanvictor** [comment]
hope you get better soon!

**Comment by TysonMN** [comment]
I got [my GitHub page site up](https://tysonmn.github.io/) ~(though with some broken asset links)~.

@loreanvictor, can you make `http://tysonwilliams.coding.blog/` point to `https://tysonmn.github.io/`?

**Comment by im-coder-lg** [comment]
@loreanvictor I will use GitHub Pages only - don't waste your money on my domain.

**Comment by loreanvictor** [comment]
@TysonMN [yes](https://mxtoolbox.com/SuperTool.aspx?action=cname%3atysonwilliams.coding.blog&run=toolpage). You now need to also set the custom domain name on GitHub Pages to tysonwilliams.coding.blog and all should work fine (I'm not 100% sure about the assets though, if you run into further issues I think a setup [similar to mine](https://github.com/loreanvictor/techblog/blob/249be4b3a795b8143f9ab32a32c79ee1738cfd14/.codedoc/config.ts#L13-L18) should do the trick).

@im-coder-lg an extra subdomain is basically free for me, so feel free to just post the repo you want the address to point to (actually, the account is important, you can then point it to any repo you want) and I'll gladly config the DNS accordingly.

**Comment by TysonMN** [comment]
Perfect.  I followed [these instructions](https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/managing-a-custom-domain-for-your-github-pages-site#configuring-a-subdomain) and it just worked :)

> I'm not 100% sure about the assets though, if you run into further issues I think a setup [similar to mine](https://github.com/loreanvictor/techblog/blob/249be4b3a795b8143f9ab32a32c79ee1738cfd14/.codedoc/config.ts#L13-L18) should do the trick

Yes.  I was looking at your commits.  I solved my asset problem.  Anyone interested in how I solved it can look at my commits [starting here](https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/managing-a-custom-domain-for-your-github-pages-site#configuring-a-subdomain).

**Comment by uloco** [comment]
I'm running another portfolio on github pages so that is not an option for me. Would it be possible to host the blog on vercel? What do I need to make this work?
